staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -59,6 +61,15 @@ const (
  * the metric stability policy.
  */
 var (
+	// these are the valid content types which we will report in request metrics. Any other content types
+	// will be aggregated under 'unknown'
+	validMetricContentTypes = utilsets.NewString("application/vnd.kubernetes.protobuf", "application/json", "application/vnd.kubernetes.protobuf;stream=watch", "text/plain")
```

Comment:
Isn't yaml also accepted?

Reply:
And json patch?

Can we enumerate the types we're currently recording?

Reply:
What do you mean? Figure this set out at runtime?

Reply:
I don't actually have full list, I was hoping people would chime in with ones that I was possibly missing. These are the unique contentTypes I pulled from a running cluster.

Reply:
Updated with a more complete list. Also added validation logic to ensure that the content-type is actually RFC compliant.